api/src/pcapi/core/educational/repository.py

```diff
@@ -730,6 +730,14 @@ def get_collective_offer_by_id(offer_id: int) -> educational_models.CollectiveOf
         raise educational_exceptions.CollectiveOfferNotFound()
 
 
+def get_offerer_ids_from_collective_offers_ids(offers_ids: list[int]) -> set[int]:
+    query = db.session.query(offerers_models.Offerer)
```

Comment:
```suggestion
    query = db.session.query(offerers_models.Offerer.id)
```
Pas besoin de remonter tout l'objet, l'id suffit (et ça va plus vite) et ce serait plus raccord avec le type déclaré